src/System.Private.CoreLib/src/System/DateTime.cs

```diff
@@ -925,7 +1087,7 @@ public static bool IsLeapYear(int year)
         // 
         public static DateTime Parse(String s)
         {
-            return (FormatProvider.ParseDateTime(s, null, DateTimeStyles.None));
```

Comment:
>FormatProvider.ParseDateTime(s, null, DateTimeStyles.None)); [](start = 20, length = 60)

It'll be good if we can keep the calls from FormatProvider. this mainly done before to factor the dependency on the globalization. 

CC @weshaggard